mava/utils/make_env.py

```diff
@@ -100,8 +100,9 @@ def make_jaxmarl_env(env_name: str, config: Dict) -> Tuple[Environment, Environm
     env = JaxMarlWrapper(jaxmarl.make(env_name, **kwargs))
     eval_env = JaxMarlWrapper(jaxmarl.make(env_name, **kwargs))
 
-    env = add_optional_wrappers(env, config)
-    eval_env = add_optional_wrappers(eval_env, config)
+    if config["system"]["add_agent_id"]:
+        env = AgentIDWrapper(env, config["system"]["add_global_state"])
+        eval_env = AgentIDWrapper(eval_env, config["system"]["add_global_state"])
```

Comment:
If you like the changes suggested below then just make these both `True` and put a comment that JaxMarlEnvs always have global state